packages/scheduler/lib/db/migrations/20250509205800_tasks_retry_key_owner_key.ts

```diff
@@ -0,0 +1,24 @@
+import type { Knex } from 'knex';
+import { TASKS_TABLE } from '../../models/tasks.js';
+
+export const config = {
+    transaction: false
+};
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.raw(
+        `ALTER TABLE ${TASKS_TABLE}
+        ADD COLUMN IF NOT EXISTS retry_key UUID,
+        ADD COLUMN IF NOT EXISTS owner_key VARCHAR(64);
```

Comment:
I picked a length of 64 arbitrarily. That's plenty of space to have meaningful value. We could even make it smaller